net4j/com.b2international.snowowl.eventbus/src/com/b2international/snowowl/eventbus/net4j/EventBusNet4jUtil.java

```diff
@@ -33,12 +34,16 @@
 	 * network.
 	 * 
 	 * @param container
+	 * @param b 
```

Comment:
Please change `b` to `gzipEnabled`; `gzipEnabled` in turn can also be shortened to `gzip`.